app/src/main/java/org/greenstand/android/TreeTracker/signup/SignupViewModel.kt

```diff
@@ -10,10 +10,17 @@ import org.greenstand.android.TreeTracker.utilities.Validation
 // Dequeue breaks equals so state will not be updated when navigating
 data class SignUpState(
     val name: String? = null,
+    val email: String? = null,
+    val phone: String? = null,
     val organization: String? = null,
     val photoPath: String? = null,
     val credential: Credential = Credential.Email(),
-)
+   )
+data class CredentialState(
+    val nameEntryStage: Boolean = false,
+    val isEmailValid: Boolean = false,
+    val isPhoneValid: Boolean = false,
+    )
```

Comment:
Each viewmodel should have a single state object. You can move these 3 fields into the SignUpState object.